### ISSUE TITLE
Could be possible to use masurca with IonTorrent + PacBio ?

### ISSUE THREAD

**Issue by eandresleon** [alekseyzimin/masurca#171]
Hi, We are working woth a parasite and we have data from a IonTorrent and a PacBio (So, no illumina reads). Could be possible to use masurca with IonTorrent + PacBio ?

Thanks in advance

**Comment by alekseyzimin** [comment]
No, unfortunately not. You have to have Illumina paired end reads.
